stats/includes/member.inc

```diff
@@ -309,12 +309,6 @@ function showForumProfile($username)
             "https://www.twitter.com/" . @$bb_user["twitter"],
             "_new",
         ],
-        [
-            "Google+",
-            @$bb_user["googleplus"],
```

Comment:
is it worth removing it from `$interested_columns` in forum_interface.inc and the bb table?

Reply:
My inclination is to leave it in both places until phpBB stops supporting it in 4.x. I removed it primarily because it was easier than fixing the "don't do string stuff to a Null value" for a no-longer-around social media site.